### ISSUE TITLE
TAB indentation removed for some simple "if" statement

### ISSUE THREAD

**Issue by arnaud-carre** [slynch8/10x#2481]
sometimes when I add some { } over some big functions, 10x removes the TAB indenting "some" simple "if" like

if ( a )
\tfoo

becomes

if ( a )
foo

I notice that when looking at the diff of some big refactor, I didn't have small repro yet. Did you know about this behavior?

**Comment by slynch8** [comment]
No, I've not seen that. I often do the same and I've never noticed it happen. Let me know if you find a repro.
